frontend/src/views/shop/DetailShop.vue

```diff
@@ -0,0 +1,903 @@
+                        {{ formatDateTime(row.nextExecutionTime) }}
+                    </template>
+
+                    <template #cell-status="{ row }">
+                        <span
+                            v-if="row.status === 'scheduled' && !row.overdue"
+                            class="pill pill-success"
+                        >
+                            <icon-fa6-solid:check />
+                            {{ row.status }}
+                        </span>
+                        <span
+                            v-else-if="row.status === 'queued' || row.status === 'running' && !row.overdue"
+                            class="pill pill-info"
+                        >
+                            <icon-fa6-solid:rotate />
+                            {{ row.status }}
+                        </span>
+                        <span
+                            v-else-if="
+                                row.status === 'scheduled' ||
+                                    row.status === 'queued' ||
+                                    row.status === 'running' &&
+                                    row.overdue"
+                            class="pill pill-warning"
+                        >
+                            <icon-fa6-solid:info class="align-[-0.1em]" />
+                            {{ row.status }}
+                        </span>
+                        <span
+                            v-else-if="row.status === 'inactive'"
+                            class="pill"
+                        >
+                            <icon-fa6-solid:pause />
+                            {{ row.status }}
+                        </span>
+                        <span
+                            v-else
+                            class="pill pill-error"
+                        >
+                            <icon-fa6-solid:xmark />
+                            {{ row.status }}
+                        </span>
+                    </template>
+
+                    <template #cell-actions="{ row }">
+                        <span
+                            class="cursor-pointer opacity-25 hover:opacity-100 tooltip-position-left"
+                            data-tooltip="Re-schedule task"
+                            @click="onReScheduleTask(row.id)"
+                        >
+                            <icon-fa6-solid:arrow-rotate-right class="text-base leading-3" />
+                        </span>
+                    </template>
+                </data-table>
+            </template>
+
+            <template #panel-queue>
+                <data-table
+                    :columns="[
+                        { key: 'name', name: 'Name', sortable: true },
+                        { key: 'size', name: 'Size', sortable: true },
+                    ]"
+                    :data="shopStore.shop.queueInfo || []"
+                />
+            </template>
+
+            <template #panel-pagespeed>
```

Comment:
Change to tab page speed get an JS error
<img width="1259" alt="Bildschirmfoto 2024-03-09 um 19 03 16" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/be4768ee-17bb-497c-927a-cec3a6ea7ce8">
<img width="782" alt="Bildschirmfoto 2024-03-09 um 19 03 37" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/d06e87f5-0177-4a51-a37b-3cc976ef07b5">

Reply:
fixed this too